packages/website/lib/faqContent.js

```diff
@@ -48,8 +48,11 @@ const faqContent = {
   ),
   nftSizeRestrictions: (
     <p className="lh-copy white mb4">
-      NFT.Storage can store NFTs up to <strong>31GiB </strong>
-      in size!
+      NFT.Storage accepts storage requests up to <strong>31GiB</strong> in size! Each upload can include a single file or a directory of files. (If you are using the HTTP API, you'll need to do some manual splitting for files over 100MB. See the{' '}
```

Comment:
@mishmosh apparently you need to use `&apos;` instead of `'` here.

```suggestion
      NFT.Storage accepts storage requests up to <strong>31GiB</strong> in size! Each upload can include a single file or a directory of files. (If you are using the HTTP API, you&apos;ll need to do some manual splitting for files over 100MB. See the{' '}
```